fri/src/prover.rs

```diff
@@ -168,3 +171,7 @@ struct CommitPhaseResult<FC: FriConfig> {
     data: Vec<<FC::CommitPhaseMmcs as Mmcs<FC::Challenge>>::ProverData>,
     final_poly: FC::Challenge,
 }
+
+pub fn generate_pow_witness<FC: FriConfig>(config: &FC, challenger: &FC::Challenger) -> FC::Val {
```

Comment:
nit: I'd just inline since the line in here is a bit shorter than the line to call this